RELEASE-NOTES.txt

```diff
@@ -3,6 +3,7 @@ Unreleased
 * [*] Remove third-party dependency on react-native-hsv-color-picker [https://github.com/WordPress/gutenberg/pull/53329]
 * [*] Search Control - Prevent calling TextInput's methods when undefined [https://github.com/wordpress-mobile/gutenberg-mobile/pull/6090]
 * [**] Add basic support to view, relocate, and remove the Jetpack Paywall block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/6076]
+* [**] [RNMobile] Add block outline to all Social Link blocks when selected [https://github.com/WordPress/gutenberg/pull/54011]
```

Comment:
IMO the `[RNMobile]` tag isn't relevant or helpful in release notes, which ultimately are user-facing. All changes in these release notes relate to the mobile editor. 
```suggestion
* [**] Add block outline to all Social Link blocks when selected [https://github.com/WordPress/gutenberg/pull/54011]
```